src/content/en/updates/2017/04/chrome-59-deprecations.md

```diff
@@ -0,0 +1,134 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 58 to help you plan.
+
+{# wf_updated_on: 2017-05-01 #}
+{# wf_published_on: 2017-05-01 #}
+{# wf_tags: deprecations,removals,chrome59 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 58 to help you plan. #}
+
+# Deprecations and Removals in Chrome 59 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 59,
+which is in beta as of April 27. This list is subject to change at any time.
+
+## Remove features from WebVR that are not in the revised spec
+
+The current implementation of WebVR, originally implemented in Chrome 52,
+contained several methods and properties that will not be in the final spec.
+Deprecation messages were added for these features for the
+[Origin Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md)
+that started in Chrome 56 and are now being removed. These features include:
+
+- `VRDisplay.getPose()`
+- `VRDisplay.resetPose()`
+- `VRDisplay.isConnected`
+- `VRDisplayCapabilities.hasOrientation`
+- `VREyeParameters.fieldOfView`
+
+[Intent to Experiment](https://groups.google.com/a/chromium.org/d/topic/blink-dev/zGAzqfi0e00/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/4532810371039232) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=706561&desc=2) &#124;
+[Origin Trial Results so Far](https://groups.google.com/a/chromium.org/d/topic/blink-dev/c41q3tyCBJE/discussion)
+
+## Remove "on-demand" value for hover/any-hover media queries
+
+The “on-demand” value for hover/any-hover media queries was removed from the
+spec about a year ago. Consequently, these media queries are removed in Chrome
+59.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/-sTmxMpl6iI/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/4719452646014976) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=654861)
+
+
+## Remove remote and readonly members of MediaStreamTrack
+
+In Chrome 48 the `MediaStreamTrack.remote` and `MediaStreamTrack.readonly`
+attributes were added in support of the
```

Comment:
Meta comment, because I haven't reviewed many of these and I don't know what your usual usage is, but to an ordinary JavaScript developer these would be properties not attributes.